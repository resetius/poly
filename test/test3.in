^ + + x y * 2 z 5