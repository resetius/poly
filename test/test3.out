0,0,5,32
0,1,4,80
0,2,3,80
0,3,2,40
0,4,1,10
0,5,0,1
1,0,4,80
1,1,3,160
1,2,2,120
1,3,1,40
1,4,0,5
2,0,3,80
2,1,2,120
2,2,1,60
2,3,0,10
3,0,2,40
3,1,1,40
3,2,0,10
4,0,1,10
4,1,0,5
5,0,0,1